docs/tutorial.md

```diff
@@ -783,3 +791,25 @@ pod/memcached-sample-6c765df685-mfqnz                      1/1     Running   0
 
 If you modify the size field of the `memcached-sample.yaml` and re-apply it. The
 operator will trigger a reconcile and adjust the sample pods to the size given.
+
+### Deploy your Operator with OLM
+First, install [OLM](https://sdk.operatorframework.io/docs/olm-integration/tutorial-bundle/#enabling-olm):
+
+```
+operator-sdk olm install
+```
+
+Bundle your operator, then build and push the bundle image. The [bundle](https://github.com/operator-framework/operator-registry/blob/v1.16.1/docs/design/operator-bundle.md#operator-bundle) target generates a bundle in the bundle directory containing manifests and metadata defining your operator. bundle-build and bundle-push build and push a bundle image defined by bundle.Dockerfile.
+
+```
+make bundle bundle-build bundle-push
```

Comment:
Do we not have to specify the bundle image via `BUNDLE_IMG`? Or does will it magically create `memcached-operator-bundle`?

Reply:
Yes, either User needs to pass it or modify the makefile. The below link is showing how to pass it
https://sdk.operatorframework.io/docs/olm-integration/quickstart-bundle/#steps and I added that in the tutorial.

Reply:
Maybe we can ask User to export below variables

```
$ export USERNAME=<container-registry-username>
$ export VERSION=0.0.1
$ export IMG=docker.io/$USERNAME/memcached-operator:v$VERSION // location where your operator image is hosted
$ export BUNDLE_IMG=docker.io/$USERNAME/memcached-operator-bundle:v$VERSION // location where your bundle will be hosted
```

I will add it so that it will help User.